contracts/EnsCounting.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/governance/Governor.sol";
+
+/**
+ * @dev Based on GovernorCountingSimple, and added AgainstWithoutBond VoteType logic
+ */
+abstract contract ENSCounting is Governor {
+
+    enum VoteType {
+        Against,
+        For,
+        Abstain,
+        AgainstWithoutBond
```

Comment:
Implementing it this way, as a fourth VoteType option, will work, and it might be the best option, but it will break compatibility with existing governance clients. A backwards-compatible way to implement this would be to add an optional flag to the `params` argument of `voteWithParams` that is interpreted as signaling desire to vote for bond forfeiture.